.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+
+      - name: Build package
+        run: pipx run build
+
+      - name: Publish to Test PyPi
+        if: ${{ startsWith(github.ref, 'refs/tags') }}
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
```

Comment:
@pseewald If you can generate the tokens from PyPi Test and PyPi and store them as secrets with the name `TEST_PYPI_API_TOKEN`  and `PYPI_API_TOKEN`, then this action can be used for publishing i.e. no need to use twine

Reply:
I'm not familiar with that process. Alternatively, we move fprettify soon to @fortran-lang, then you can take care of this as well (if you agree)?

Reply:
This is just a nice way to publish and upload packages to PyPi  via the GitHub UI. It calls `twine` in the background so it needs a valid PyPi token to successfully upload the package to [`pseewald`](https://pypi.org/user/pseewald/)

This is a similar example to how the Language Server does it: https://github.com/gnikit/fortls/blob/25b793c224ce888aa533928e75386fba565d4942/.github/workflows/python-publish.yml#L34-L46

I am happy for fprettify to move to fortran-lang. Still in that case the publishing issue would persist. You would have to add the [Fortran-lang PyPi](https://pypi.org/user/fortran-lang/) account as a maintainer or owner to be able to publish the package on PyPi.